hadoop-ozone/dist/src/main/compose/upgrade/test.sh

```diff
@@ -26,7 +26,6 @@ export COMPOSE_DIR
 export OZONE_VOLUME
 
 mkdir -p "${OZONE_VOLUME}"/{dn1,dn2,dn3,om,recon,s3g,scm}
-mkdir -p "${OZONE_VOLUME}/debug"
```

Comment:
`/debug` is leftover means it is unused now?

Reply:
Yes.  I had added it for a `debug` container while trying to find out the cause of permission problems in GitHub Actions (it was working fine locally).  I [removed the container](https://github.com/apache/hadoop-ozone/pull/1142/commits/9dbbcbb7e1719a5bba94533e49674556d4b1e362) in the final version, but forgot about the directory.